Airports/EGJJ/Ownership.txt

```diff
@@ -30,18 +30,59 @@ ALTOWNER:Mentoring TWR/APP:JJAM:JJA:JM:J:LW:LWN:LWC:LWS:L
 BORDER:JJAPP
 ARRAPT:EGJJ
 
-SECTOR:JJCTR:0:19500
+SECTOR:JJAPP-M:0:3000
+OWNER:JJR:JJA:J:JS:LW:LWN:LWC:LWS:L
+ALTOWNER:Observing LON_W_CTR:JJA:J:O:LM:LW:LWN:LWC:LWS:L
+ALTOWNER:Observing London FIR:O:LM:JJA:J:LW:LWN:LWC:LWS:L
+ALTOWNER:Mentoring TWR/APP:JJAM:JJA:JM:J:LW:LWN:LWC:LWS:L
+BORDER:JJAPP-M
+
+SECTOR:JJAPP-S:0:5000
+OWNER:JJR:JJA:J:JS:LW:LWN:LWC:LWS:L
+ALTOWNER:Observing LON_W_CTR:JJA:J:O:LM:LW:LWN:LWC:LWS:L
+ALTOWNER:Observing London FIR:O:LM:JJA:J:LW:LWN:LWC:LWS:L
+ALTOWNER:Mentoring TWR/APP:JJAM:JJA:JM:J:LW:LWN:LWC:LWS:L
+BORDER:JJAPP-S
+
+
+SECTOR:JJCTR-N-ORTAC:0:19500
 OWNER:J:JS:LW:LWN:LWC:LWS:L
 ALTOWNER:Observing LON_W_CTR:J:O:LM:LW:LWN:LWC:LWS:L
 ALTOWNER:Observing London FIR:O:LM:J:LW:LWN:LWC:LWS:L
 ALTOWNER:Mentoring TWR/APP:JM:J:LW:LWN:LWC:LWS:L
-BORDER:JJCTR-S6:JJCTR-LS:JJCTR-LFRR:JJCTR-SPLIT
+BORDER:JJCTR-N-ORTAC-S:JJCTR-N-ORTAC-LONS:JJCTR-N-ORTAC-LFRR:JJCTR-N-ORTAC-N
 ARRAPT:EGJB:EGJJ
 
-SECTOR:JJCTR-S:0:19500
+SECTOR:JJCTR-M-ORTAC:3000:19500
+OWNER:J:JS:LW:LWN:LWC:LWS:L
+ALTOWNER:Observing LON_W_CTR:J:O:LM:LW:LWN:LWC:LWS:L
+ALTOWNER:Observing London FIR:O:LM:J:LW:LWN:LWC:LWS:L
+ALTOWNER:Mentoring TWR/APP:JM:J:LW:LWN:LWC:LWS:L
+BORDER:JJCTR-M-ORTAC-S:JJCTR-M-SPLT:JJCTR-M-ORTAC-N
```

Comment:
Error: Not closed border lines JJCTR-N-ORTAC/JJCTR-N-ORTAC-S/JJCTR-N-ORTAC-N

This error means that sector JJCTR-N-ORTAC is not closed properly, because JJCTR-N-ORTAC-S and JJCTR-N-ORTAC-N do not share a coordinate